src/main/resources/public/js/view.js

```diff
@@ -169,22 +174,44 @@ function drawBot(b, scale, xOffset, yOffset) {
         circle3.drawCircle(0, 0, scale/10);
         circle3.endFill();
 
-    circle2.x = cx+botradius*Math.cos(b.angle+Math.PI/6);
-    circle2.y = cy+botradius*Math.sin(b.angle+Math.PI/6);
+    circle2.x = cx+size*Math.cos(b.angle+Math.PI/6);
```

Comment:
in the future it would be ideal to generate these from data on the bot rather than assuming the bot has sensors